script/update-docs.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+#
+# This script generate the README.md in helm/prometheus-operator-app helm chart
```

Comment:
I think we want to generate the CONTRIBUTING.md, right ?

Reply:
No

Reply:
We generate the helm/prometheus-operator-app/README.md

Reply:
I am confused now. The `helm/prometheus-operator-app/README.md` is removed in this PR. So what is this script for ?

Reply:
I have to regenerate

Reply:
Did you regenerate the helm/prometheus-operator-app/README.md?

Reply:
I did